packages/rulesets/generated/spectral/az-common.js

```diff
@@ -955,6 +961,8 @@ const ruleset = {
             description: "The value of the 'description' property must be descriptive. It cannot be spaces or empty description.",
             message: "'{{property}}' parameter lacks 'description' property. Consider adding a 'description' element. Accurate description is essential for maintaining reference documentation.",
             severity: "error",
+            disableForTypeSpec: true,
+            disableForTypeSpecReason: "TODO",
```

Comment:
```suggestion
            disableForTypeSpecReason: "Covered by TSP's '@azure-tools/typespec-azure-core/documentation-required' rule.",
```

I believe this is the same rule

Reply:
I'm curious how "ParameterDescriptionRequired" is not duplicative of "ParameterDescription"?  Either way, they can both be disabled.